config/repos.yml

```diff
@@ -51,6 +51,59 @@ master:
   manageiq-v2v:
   manageiq_docs:
   ui-components:
+jansa:
+  amazon_ssa_support:
+  container-httpd:
+    has_real_releases: true
+  container-memcached:
+    has_real_releases: true
+  container-postgresql:
+    has_real_releases: true
+  container-ruby:
+    has_real_releases: true
+  dbus_api_service:
+    has_real_releases: true
+  httpd_configmap_generator:
+    has_real_releases: true
+  manageiq:
+    has_rake_release: true
+  manageiq-api:
+  manageiq-appliance:
+  manageiq-appliance-build:
+  manageiq-appliance_console:
+    has_real_releases: true
+  manageiq-automation_engine:
+  manageiq-consumption:
+  manageiq-content:
+    has_rake_release: true
+  manageiq-decorators:
+  manageiq-gems-pending:
+  manageiq-graphql:
+  manageiq-pods:
+  manageiq-providers-amazon:
+  manageiq-providers-ansible_tower:
+  manageiq-providers-azure:
+  manageiq-providers-azure_stack:
+  manageiq-providers-foreman:
+  manageiq-providers-google:
+  manageiq-providers-kubernetes:
+  manageiq-providers-kubevirt:
+  manageiq-providers-lenovo:
+  manageiq-providers-nuage:
+  manageiq-providers-openshift:
+  manageiq-providers-openstack:
+  manageiq-providers-ovirt:
+  manageiq-providers-redfish:
+  manageiq-providers-scvmm:
+  manageiq-providers-vmware:
+  manageiq-schema:
+  manageiq-smartstate:
+    has_real_releases: true
+  manageiq-ui-classic:
+  manageiq-ui-service:
+  manageiq-v2v:
+  manageiq_docs:
+  ui-components:
```

Comment:
Can we add `has_real_releases: true` to ui-components? 

ui-components is different from other repos where we want to create branches, but we don't tag as it's a released package.  Wonder if we should change this so it will follow the normal "with" or "without" release process, not in between.  Or at least make a different option in this yaml?

Related PR: https://github.com/ManageIQ/manageiq-release/pull/95

Reply:
@simaishi Done (I didn't do it for master, because it wasn't done for master previously anyway)